### ISSUE TITLE
undefined: termios.SetWinSize

### ISSUE THREAD

**Issue by tartavull** [charmbracelet/ssh#28]
```
# github.com/charmbracelet/ssh
/Users/ignaciotartavull/go/pkg/mod/github.com/charmbracelet/ssh@v0.0.0-20240515141028-546b2ee33a4d/pty_unix.go:57:18: undefined: termios.SetWinSize
/Users/ignaciotartavull/go/pkg/mod/github.com/charmbracelet/ssh@v0.0.0-20240515141028-546b2ee33a4d/pty_unix.go:144:17: undefined: termios.SetWinSize
```

I started getting this error in the lastest version :/

**Comment by maaslalani** [comment]
I am also receiving this error in [`huh`](https://github.com/charmbracelet/huh)

<img width="1526" alt="image" src="https://github.com/charmbracelet/ssh/assets/42545625/b4e101f4-98c7-4dc6-b400-e6cbe6dc9e58">

**Comment by antoniomika** [comment]
Can be closed as fixed by [04041eb](https://github.com/charmbracelet/ssh/commit/04041eb117c779e1b877bc8d47b05a184456ce49)
